transform/models/marts/fact_commits_monthly.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+  d.date_month,
+  dr.repo_name,
+  COUNT(*) as count_stars,
+  LAG(count_stars,1) OVER(PARTITION BY dr.repo_name ORDER BY d.date_month)as last_month_count_stars,
```

Comment:
Naming: Should be `last_month_count_commits`

Reply:
Gotchaa